cmd/csi-provisioner/csi-provisioner.go

```diff
@@ -64,7 +64,10 @@ var (
 	operationTimeout     = flag.Duration("timeout", 10*time.Second, "Timeout for waiting for creation or deletion of a volume")
 	_                    = deprecatedflags.Add("provisioner")
 
-	enableLeaderElection    = flag.Bool("enable-leader-election", false, "Enables leader election. If leader election is enabled, additional RBAC rules are required. Please refer to the Kubernetes CSI documentation for instructions on setting up these RBAC rules.")
+	_ = deprecatedflags.Add("enable-leader-election")
+
+	enableLeaderElection = flag.Bool("leader-election", false, "Enables leader election. If leader election is enabled, additional RBAC rules are required. Please refer to the Kubernetes CSI documentation for instructions on setting up these RBAC rules.")
```

Comment:
This is a breaking change that requires bump to v2.0.
@msau42, can we expect that the next release will be 2.0?

@Madhu-1, depending on the answer to above question, would it be possible to support both `enable-leader-election` and `leader-election` at the same time until we're ready for v2.0? It's OK to throw error if both are specified.

Reply:
@jsafrane yes I can do that, will wait for @msau42 response

Reply:
I just released a 1.6, and mentioned this in the deprecation notes. We can proceed and just break it.